client/src/InfoBase/bootstrapper.js

```diff
@@ -4,8 +4,7 @@ import "whatwg-fetch";
 
 import "../common_css/common_css_index.side-effects.js";
 
-import "./inject_app_globals.side-effects.js";
-import "../handlebars/initialize_handlebars.side-effects.js";
+import "../handlebars/register_helpers.side-effects.js";
```

Comment:
Ah okay, I was essentially trying to remove side effects altogether, to put all registerHelpers inside `text` or `format` but this still removes globals !

Reply:
I'd like to still untangle it more. There's too much of a mess with there being, for example, helpers that need access to glossary while glossary itself needs access to text maker. We'd have to come up with our own way to manage the helper dependencies of yaml files or something though, so way too out of scope for here.